cns/ipampool/metrics.go

```diff
@@ -5,66 +5,81 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/metrics"
 )
 
+const (
+	subnetLabel     = "Subnet"
+	subnetCIDRLabel = "SubnetCIDR"
```

Comment:
snake-case [for consistency](https://github.com/paulgmiller/azure-container-networking/blob/17258152314ee33986afbd8c011343d72d4511b1/cns/restserver/metrics.go#L39).